docs/source/tutorials/query_dataset.rst

```diff
@@ -0,0 +1,134 @@
+- How to use a jupyter notebook or python interpreter
+
+Setup Environment
+=================
+
+1. Setup python environment
+
+from a terminal run:
+
+.. code-block:: bash
+
+   $ module load python  # only if running on kepler
+
+   $ python -m venv dsgrid-tutorial
+   $ source dsgrid-tutorial/bin/activate
+
+2. Install duckdb and pandas
+
+.. code-block:: bash
+
+   $ pip install duckdb
+   $ pip install pandas
+
+
+Load data from Kestrel
+======================
+
+1. Enter a python interpreter
+
+.. code-block:: bash
+
+   $ python
+
+2. Load .parquet files from Kestrel into a table
+
+.. code-block:: python
+
+   import duckdb
+
+   tablename = "tbl"
+   data_dir = "/datasets/dsgrid/dsgrid-tempo-v2022"
+   dataset_name = "state_level_simplified"
+   filepath = f"{data_dir}/{dataset_name}"
+
+   duckdb.sql(f"""CREATE TABLE {tablename} AS SELECT * 
+                FROM read_parquet('{filepath}/table.parquet/**/*.parquet',
+                hive_partitioning=true, hive_types_autocast=false""")
+
+   duckdb.sql(f"DESCRIBE {tablename}")  # shows columns and types
+   duckdb.sql(f"SELECT * FROM {tablename} LIMIT 5").df()  # shows first 5 rows
+
+
+Load data from OEDI
+===================
+
+1. Enter python interpreter the same as above
+   
+2. Load .parquet files from OEDI into a table
+   This step is identical to the previous loading step, except the file path is different
+
+.. code-block:: python
+
+   import duckdb
+
+   tablename = "tbl"
+   data_dir = "s3://nrel-pds-dsgrid/tempo/tempo-2022/v1.0.0/"
+   dataset_name = "state_level_simplified"
+   filepath = f"{data_dir}/{dataset_name}"
+
+   duckdb.sql(f"""CREATE TABLE {tablename} AS SELECT * 
+                FROM read_parquet('{filepath}/table.parquet/**/*.parquet',
+                hive_partitioning=true, hive_types_autocast=false""")
+
+   duckdb.sql(f"DESCRIBE {tablename}")  # shows columns and types
+   duckdb.sql(f"SELECT * FROM {tablename} LIMIT 5").df()  # shows first 5 rows
+
+Filter data with duckdb
+=======================
+
+One of the main advantages to using duckdb is the ability to filter
+data while loading. If a table is created with a filter, duckdb will
+not have to read all of the data to generate the requested table. This
+can make queries much more efficient.
+
+Using the same tablename and filepath from the sections above
+
+.. code-block:: python
+
+   duckdb.sql("""CREATE TABLE {tablename} AS SELECT *
+                FROM read_parquet('{filepath}/table.parquet/**/*.parquet',
+                hive_partitioning=true, hive_types_autocast=false
+                WHERE state='MI' AND scenario='efs_high_ldv'
+             """)
+                
+Export Data
+===========
+
+1. Create a pandas dataframe after loading, and possibly filtering, from the previous steps
+
+.. code-block:: python
+
+   dataframe = duckdb.sql("SELECT * FROM {tablename}").df()
+
+2. Export dataframe to csv after createing dataframe
+
+.. code-block:: python
+
+   dataframe.to_csv('~/mydata.csv')
```

Comment:
The `~` won't work. You could use environment variables or something like this:
```python
from pathlib import Path
dataframe.to_csv(Path.home() / 'mydata.csv')
```

Reply:
for an example would it make sense to just write the csv locally with a relative path 'mydata.csv'?

Reply:
sure